packages/fx-core/resource/package.nls.json

```diff
@@ -932,17 +932,15 @@
   "driver.prerequisite.summary.func.installed": "Azure Functions Core Tools is installed.",
   "driver.prerequisite.summary.dotnet.installedWithPath": ".NET Core SDK is installed at %s.",
   "driver.prerequisite.summary.dotnet.installed": ".NET Core SDK is installed.",
-  "driver.env.addEnvironmentVariable.description": "Generate environment variables to project environment file",
-  "driver.env.addEnvironmentVariable.summary": "The environment variables have been generated successfully to the .env file of '%s' environment.",
   "driver.file.createOrUpdateEnvironmentFile.description": "Create or update variables to environment file.",
   "driver.file.createOrUpdateEnvironmentFile.summary": "The variables have been generated successfully to %s.",
   "driver.file.createOrUpdateJsonFile.description": "Create or update appsettings to JSON file.",
   "driver.file.createOrUpdateJsonFile.summary": "The appsettings have been generated successfully to %s.",
   "driver.file.description": "generates environment variables to .env file",
   "driver.file.summary.default": "The environment variables has been generated successfully to the .env file of '%s' environment.",
   "driver.file.summary.withTarget": "The environment variables has been generated successfully to %s.",
-  "driver.file.progressBar.title": "file/updateEnv",
-  "driver.file.progressBar.generate": "Generating the environment variables to .env file.",
+  "driver.file.progressBar.appsettings": "Generating appsettings to file...",
```

Comment:
recommend to remove "to file" as it is for progress bar, can make it short. same for the next line.

Reply:
Updated

Reply:
@sffamily  I've updated the strings. Could you please help review again?